pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2076,6 +2076,15 @@ var _ = Describe("Converter", func() {
 		)
 	})
 
+	It("should not include serial entry in sysinfo when firmware.serial is not set", func() {
+		vmi := libvmi.New()
+		v1.SetObjectDefaults_VirtualMachineInstance(vmi)
+		domain := vmiToDomain(vmi, &ConverterContext{Architecture: archconverter.NewConverter(runtime.GOARCH), AllowEmulation: true})
+		Expect(domain.Spec.SysInfo.System).ToNot(ContainElement(HaveField("Name", Equal("serial"))),
+			"serial entry should not be present in sysinfo",
+		)
+	})
```

Comment:
I would prefer to test it for every arch explicitly:
```suggestion
	DescribeTable("should not include serial entry in sysinfo when firmware.serial is not set", func(arch string) {
		vmi := libvmi.New()
		v1.SetObjectDefaults_VirtualMachineInstance(vmi)
		domain := vmiToDomain(vmi, &ConverterContext{Architecture: archconverter.NewConverter(arch), AllowEmulation: true})
		Expect(domain.Spec.SysInfo.System).ToNot(ContainElement(HaveField("Name", Equal("serial"))),
			"serial entry should not be present in sysinfo",
		)
	},
		MultiArchEntry(""),
	)
```

Reply:
> but it's very cheap so I wouldn't insist

Thanks, but I do not find it cheap.

I don't know the history of `MultiArchEntry` but I find it harmful in this context. It would add complexity to a simple/stupid, duplicate runtime by four, with zero benefit. There is zero risk that a rogue firmware.serial to domxml only in one architecture.

Reply:
@fossedihelm Hi, sorry but I am a bit confused. Can you please elaborate on the multi-arch [issue](https://github.com/kubevirt/kubevirt/issues/13258)? If I will compile the project on non x86 machine, and I will run `make go-test`, not all the converter tests will run? Does the test framework implicitly filters tests based on the GO_ARCH?
Is this the reason you've created the `MultiArchEntry`?